examples/soupbintcp-client/README.md

```diff
@@ -0,0 +1,35 @@
+# Nassau SoupBinTCP Test Client
+
+Nassau SoupBinTCP Test Client is an example application implementing SoupBinTCP.
```

Comment:
Maybe the end could be something like this:
> ... implementing a SoupBinTCP client.

Likewise on the server side.

Reply:
Done